docs/changes.rst

```diff
@@ -54,21 +54,21 @@ Released on December 20th 2019.
 * Fixed creating new languages.
 * Avoid sending auditlog notifications to deleted e-mails.
 * Added support for read only strings.
-* Added support for markdown in comments.
-* Allow to place translation instructions text into project info.
+* Added support for Markdown in comments.
+* Allow placing translation instruction text in project info.
 * Add copy to clipboard for secondary languages.
 * Improved support for Mercurial.
-* Improved performance in fetching Git repositories.
+* Improved Git repository fetching performance.
 * Add search lookup for age of string.
 * Show source language for all translations.
 * Show context for nearby strings.
 * Added support for notifications on repository operations.
 * Improved translation listings.
 * Extended search capabilities.
-* Added support for automatic translation strings marked for edit.
+* Added support for automatic translation strings marked for editing.
 * Avoid sending duplicate notifications for linked component alerts.
 * Improve default merge request message.
-* Better indicate string state in Zen mode.
+* Better indicate string state in zen mode.
```

Comment:
Not sure what is better here. See also https://hosted.weblate.org/translate/weblate/master/en/?checksum=7867c65b412a1631#comments